src/commonMethods/constants.js

```diff
@@ -0,0 +1,14 @@
+export default {
+  //Plugin Names
+  monitorPlugin: 'Monitor',
+  controllerPlugin: 'Controller',
+  webKitBrowserPlugin: 'WebKitBrowser',
+
+  //Plugin states
+  activate: 'activate',
+  deactivate: 'deactivate',
+  resume: 'resumed',
+  suspend: 'suspend',
+
+  blankUrl: 'about:blank',
+}
```

Comment:
I don't _really_ see the added benefit of using constants (since key/values are pretty much the same ...)

Reply:
Advantage of using constants is "Whenever any constant name is changed, then we can make a change in the constants file itself, rather than modifying the test code"